cli/admin/src/main/java/io/pravega/cli/admin/utils/ZKHelper.java

```diff
@@ -194,7 +192,7 @@ private void createZKClient(String zkURL, String clusterName) throws ZKConnectio
         zkClient = CuratorFrameworkFactory
                 .builder()
                 .connectString(zkURL)
-                .namespace(BASE_NAMESPACE + SEPARATOR + clusterName)
+                .namespace(clusterName)
```

Comment:
This doesn't look like a compatible change.

Reply:
The hard coded value is making things inconsistent and hence we get exceptions. Removing it solves the problem.